test/test_mqtt5to3_adapter.py

```diff
@@ -397,6 +397,31 @@ def test_operation_sub_unsub(self):
 
         connection.disconnect().result(TIMEOUT)
 
+    def test_operation_null_ack(self):
+        TEST_TOPIC = '/test/topic/adapter' + str(uuid.uuid4())
+        exception_occurred = False
+
+        try:
+            client, mqtt5_callbacks = self._setup_direct_connect_mutual_tls()
+            mqtt311_callbacks = Mqtt311TestCallbacks()
+            connection = self._create_connection(client, mqtt311_callbacks)
+
+            # publish an offline packet so that the publish operation would be incomplete
+            published, packet_id = connection.publish(TEST_TOPIC, self.TEST_MSG, QoS.AT_LEAST_ONCE)
+
+            # Manually destroyed the connection so that the incomplete publish operation should fail with exception
+            del connection
+
+            published.result(TIMEOUT)
```

Comment:
This line should be wrapped in its own try-except, so no other exception could make a test succeed